### PR TITLE
feat: private route created for admin and staff

### PR DESCRIPTION
**Pull request by nurmohammad83**
Private route created for admin and staff.

### CODE REVIEW

**Comment by annur-nifty** [comment]
Very much good checking already and this is goes perfectly

**Comment by nurmohammad83** [comment]
Thank you

File: src/App.tsx [hunk 1]
```diff
@@ -44,10 +47,17 @@ const App = () => {
...
-
+            {user?.type === "ADMIN" && (
+              // Admin Accessible routes
+              <>
+                <Route path="/transactions" element={<Transactions />} />
+                <Route path="/employees" element={<Employees />} />
+                <Route
+                  path="/rooms-overview"
+                  element={<RoomBookingFinancials />}
+                />
```
Comment (annur-nifty) [inline src/App.tsx:58]: jlkjk

**Comment by nurmohammad83** [comment]
fix: employ added form reset

**Comment by nurmohammad83** [comment]
screencapture-localhost-5173-guest-lookup-2023-11-03-20_27_22
screencapture-localhost-5173-booking-details-6544b89ca6251434b00f06b0-2023-11-03-20_27_51

**Pull request closed without merging**